main.go

```diff
@@ -247,89 +175,46 @@ func run(ctx context.Context, args []string) error {
 			}
 
 			return nil
-
 		})
 		if err != nil {
-			return fmt.Errorf("failed to walk rule dir: %w", err)
+			return fmt.Errorf("failed to walk rule path: %w", err)
 		}
 	}
 
 	if err := os.WriteFile(filepath.Join(outputDir, "bootstrap"), []byte(bootstrap), 0755); err != nil {
 		return fmt.Errorf("failed to write bootstrap file: %w", err)
 	}
 
-	// Create task.md file
-	taskOutput, err := os.Create(filepath.Join(outputDir, "task.md"))
-	if err != nil {
-		return fmt.Errorf("failed to create task file: %w", err)
-	}
-	defer taskOutput.Close()
+	// Print total token count
+	fmt.Fprintf(os.Stdout, "Total estimated tokens: %d\n", totalTokens)
 
-	for _, path := range tasks {
```

Comment:
This code block is essentially the code for the "prompt" sub-command. Add it to that command.